appveyor.yml

```diff
@@ -57,7 +57,7 @@ install:
   # Upgrade to the latest version of pip to avoid it displaying warnings
   # about it being out of date.
   - pip install --disable-pip-version-check --user --upgrade pip wheel
-  - pip install tox virtualenv
+  - pip install tox==2.1.1 virtualenv
```

Comment:
It would be nice if we could work out why tox is mad at us so we can remove these pins. Any idea?